scripts/zones/Sealions_Den/IDs.lua

```diff
@@ -31,6 +35,26 @@ zones[xi.zone.SEALIONS_DEN] =
     {
         AIRSHIP_DOOR_OFFSET = 16908420,
     },
+    aWarriorsPath = {
```

Comment:
Not a stopper, but I bet we could consolidate this down to a single offset in the future

Reply:
I did it this way because I referred to these mobs using their appropriate BFID (which as you have noted probably could have been an offset value instead) I can make this adjustment if you want.

Reply:
Since it seems to be linear (and a multiplier of 3) it may be easier to maintain.  Again, not a show-stopper, but just trying to simplify things

Reply:
all 12 are sequential so for example the others in the same battle could be tenzen+1, 2, 3.  and diff between 2 tenzen's is +4 per bf id.

Reply:
this has been fixed!